data-serving/data-service/test/case.test.ts

```diff
@@ -49,35 +49,48 @@ describe('POST', () => {
     it('create without valid date should return 422', (done) => {
```

Comment:
I'd probably just drop this test for now. There's a handful of other stuff that now has validation (we'll have to figure out the testing strategy there); date is no longer required.

Reply:
oh shoot this date was meant to be required by i accidentally dropped that... adding it back. should we keep the test in that case?

Reply:
SG